clusters/ztp-policies/common/common-policies/common-vseloki.yaml

```diff
@@ -0,0 +1,72 @@
+# yamllint disable rule:line-length
+# for this to take affect, the ManagedCluster must have "vseloki: "true"" as a label
```

Comment:
Can you please explain here what is "this"? Is it this entire file or something else?

Reply:
Adjusted, see if that makes more sense?

Reply:
Looks good